package exception;

public class Sample {
	public static void main(String[] args) {
		try {
			int x = 10;
			int y = 2;
			int z = x / y;
			int[] nums=new int[5];
			nums [5]=200;
		} catch (ArithmeticException e) {
			System.out.println(e.getMessage());
			System.out.println("Something went wrong");
	
		}
	}

}
